src/components/CallView/Grid/Grid.vue

```diff
@@ -872,6 +872,10 @@ export default {
 	position: relative;
 	bottom: 0;
 	left: 0;
+
+	&:hover .grid-navigation {
+		background-color: rgba(0, 0, 0, 0.5) !important;
+	}
```

Comment:
This is the same value that this button has initially without wrapper hovering:

https://github.com/nextcloud/spreed/blob/2aabfbb04a5deea7976b74ffe9b1a21076e5d4ec/src/components/CallView/Grid/Grid.vue#L969-L971